unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/Base Layers/RenderBackgroundColorLayer.cs

```diff
@@ -26,16 +26,21 @@ static void RenderHorizonLayer(ref float timeOfTheDay, EditorToolMeasurements to
             EditorGUILayout.Separator();
 
             // Horizon Mask
-            RenderSimpleValues.RenderTexture(SkyboxEditorLiterals.texture, ref selectedConfiguration.horizonMask);
+            EditorGUILayout.LabelField("Horizon Mask", EditorStyles.boldLabel);
```

Comment:
if `RenderHorizonLayer()` is called frequently let's cache all these new strings into CONSTs.